connection.go

```diff
@@ -30,6 +30,18 @@ const (
 	sessionClientSessionKeepAlive = "client_session_keep_alive"
 )
 
+type queryTagKeyType int
+
+const (
+	_                           = iota
+	queryTagKey queryTagKeyType = iota
```

Comment:
Will defining the `iota` here cause conflicts with `RequestContext` and `AuthClaim` in `multiplex/utils/context.go`?

Reply:
I believe this is safe, because this is a different (and private!) type, even if the value might conflict. Types are considered when go checks equality, and I guess the same applies to contexts
https://play.golang.org/p/cxkLLu3Tw3v